tests/pipelines/audio_diffusion/test_audio_diffusion.py

```diff
@@ -118,7 +118,7 @@ def test_audio_diffusion(self):
         assert image.height == self.dummy_unet.sample_size[0] and image.width == self.dummy_unet.sample_size[1]
         image_slice = np.frombuffer(image.tobytes(), dtype="uint8")[:10]
         image_from_tuple_slice = np.frombuffer(image_from_tuple.tobytes(), dtype="uint8")[:10]
-        expected_slice = np.array([255, 255, 255, 0, 181, 0, 124, 0, 15, 255])
+        expected_slice = np.array([69, 255, 255, 255, 0, 0, 77, 181, 12, 127])
```

Comment:
This test should not change since we're using the original 1000 steps DDPM scheduler here.

Reply:
no we use 4 steps. please notice steps=4 in lines 109 and 114 before these rows.
I already checked nothing is changed when using 1000 as before (many test works with 1000 and not failing).

Reply:
Thanks, yes you're 100% right.